cmd/esc/cli/client/client.go

```diff
@@ -78,8 +79,10 @@ type Client interface {
 		continuationToken string,
 	) (environments []OrgEnvironment, nextToken string, err error)
 
-	// CreateEnvironment creats an environment named envName in orgName.
+	// Deprecated
```

Comment:
We should probably still keep the original docstring while this is being deprecated.

Also, nit: we should probably follow golangs deprecation format of `Deprecated: Use {x} instead.`